js/bitstamp.js

```diff
@@ -638,6 +734,80 @@ module.exports = class bitstamp extends Exchange {
         return this.parseBalance (result);
     }
 
+    async fetchTradingFee (symbol, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        let method = 'privatePostBalance';
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['pair'] = market['id'];
+            method += 'Pair';
+        }
+        const balance = await this[method] (this.extend (request, params));
+        return {
+            'info': balance,
+            'symbol': symbol,
+            'maker': balance.fee,
+            'taker': balance.fee,
+        };
+    }
+
+    praseTradingFees (balance) {
+        const result = { 'info': balance };
+        const markets = Object.keys (this.markets);
+        for (let i = 0; i < markets.length; i++) {
+            const symbol = markets[i];
+            const fee = this.parseTradingFee (balance, symbol);
+            result[symbol] = fee;
+        }
+        return result;
+    }
+
+    async fetchTradingFees (params = {}) {
+        await this.loadMarkets ();
+        const balance = await this.privatePostBalance ();
+        return this.praseTradingFees (balance);
+    }
+
+    parseFundingFees (balance) {
+        const withdraw = {};
+        const ids = Object.keys (balance);
+        for (let i = 0; i < ids.length; i++) {
+            const id = ids[i];
+            if (id.indexOf ('_withdrawal_fee') >= 0) {
+                const currencyId = id.split ('_')[0];
+                const code = this.safeCurrencyCode (currencyId);
+                withdraw[code] = this.safeFloat (balance, id);
+            }
+        }
+        return {
+            'info': balance,
+            'withdraw': withdraw,
+            'deposit': {},
+        };
+    }
+
+    async fetchFundingFees (params = {}) {
+        await this.loadMarkets ();
+        const balance = await this.privatePostBalance ();
+        return this.parseFundingFees (balance);
+    }
+
+    async fetchFees (params = {}) {
+        await this.loadMarkets ();
+        const balance = await this.privatePostBalance ();
```

Comment:
We have to propagate the `params` to the implicit API call here and everywhere throughout this file.
```JavaScript
await this.privatePostBalance ();
```
↓
```JavaScript
await this.privatePostBalance (params);
```

Reply:
Fixed.